labs/week 1 - r review and markdown intro.Rmd

```diff
@@ -0,0 +1,158 @@
+options(scipen = 999)
+```
+
+## R Markdown
+
+This is an R Markdown document. You can embed an R code chunk like this:
+
+```{r cars}
+summary(cars)
+```
+
+On the top right of the code chunk, you can press the green triangle to run every line of code in that chunk. Go ahead and try it now.
+
+Clicking the grey triangle with the green bar under it, next to the green triangle, will run every code chunk that comes before this one. 
+
+These two functions can make it easy to run and rerun large segments of code without having to highlight them specifically. 
+
+## Including Plots
+
+You can also embed plots, for example:
+
+```{r pressure, echo=FALSE}
+plot(pressure)
+```
+
+To insert your own chunk, click on the green square with a C inside in, on the top right of this pane, or type `CTRL + ALT + I`. 
+
+Inside the chunk's header, give it a title that describes what's going on. In the above chunk, the title is "pressure." Titles cannot be reused within the same RMarkdown document, and they cannot have any spaces in them. 
+
+After the title, we have options. Note that the `echo = FALSE` parameter was added to `pressure` to prevent printing of the R code that generated the plot. Clicking the gear in the top right of the chunk gives you the options for it. You can rename the chunk and tell it what output to use. 
+
+For this class, we'll be using RMarkdown for our homework assignments, so we want to see the code that you used. So tell it to "Show code and output" or simply change the `echo=FALSE` to `echo=TRUE` in the chunk's header. 
+
+## So what is R Markdown?
+
+Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>. It's also what I used to make the slides from the Bootcamp back in August. 
+
+When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. Go ahead and click the button at the top. It'll run and compile everything into a pdf for us to view. You can then find it in the files pane to open it up. It'll have the same name as this file except with a .html ending. 
+
+You can also click the drop-down arrow next to Knit and select to knit it as a PDF or as a Word Document. This requires some more software to be added in, so we'll wait a little later to do that. 
+
+### Find the Working Directory
+
+To find out where your code is being stored, run the `getwd()` command in the below chunk.
+
+```{r getwd, echo=T}
+getwd()
+```
+
+
+As we see, mine is stored in my Documents folder, then in a folder called R, which is where all my R projects are saved. Make sure your project is saved in a permanent folder so it doesn't get lost. Saving it in Downloads, for example, runs the risk of getting deleted when you clean up your computer. 
+
+If your project is not somewhere permanent, move it now and rerun the above chunk to verify the location. 
```

Comment:
I would recommend asking students to make a new directory for all there files for this class. Each homework can then be a separate folder within this directory and they can also add directories for their projects.